config/development.json

```diff
@@ -60,6 +60,7 @@
 		"domains/premium-domain-purchases": true,
 		"email-accounts/enabled": true,
 		"external-media": true,
+		"edge-cache": true,
```

Comment:
Let's use the team prefix and full project name as we did for other projects so it would be `yolo/edge-cache-i1`.